charts/incubator/dendrite/Chart.yaml

```diff
@@ -82,6 +82,8 @@ dependencies:
 annotations:
   artifacthub.io/changes: |-
     - kind: changed
+      description: Corrected postgres persistence definition and bumped chart version
+    - kind: added
       description: Upgraded `common` chart dependency to version 4.5.0
```

Comment:
Please remove the "old" changes, since those are not relevant for this specific version :)

Reply:
Sure thing, will get it updated as soon as I am back on my computer.

Reply:
Updated.